packages/core/contracts-ovm/insured-bridge/implementation/BridgePoolFactory.sol

```diff
@@ -47,14 +55,17 @@ contract BridgePoolFactory is BridgePoolFactoryInterface, Ownable, OVM_CrossDoma
         _;
     }
 
+    // TODO: Switch to hardcoded OVM_L1CrossDomainMessenger: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/deployments/README.md
```

Comment:
I _think_ we might want to leave this parameterizable (and potentially changeable) within the contract. The reason being is right now they use a proxy for this on mainnet which could potentially change. perhaps we want to move to the underlying contract (not proxy) or there is a future pattern that requires a new address. 

Also, the proxy used on mainnet is _not_ exported anywhere within any const libs or anything like that so it becomes a bit difficult to hardcode it.